### ISSUE TITLE
Provide a way to tune jackson mapper

### ISSUE THREAD

**Issue by yarulan** [lukas-krecan/JsonUnit#156]
The library uses jackson to convert objects to json. I use [vavr](http://www.vavr.io/) collections in my classes, therefore I need to add [vavr support](https://github.com/vavr-io/vavr-jackson) to jackson.
```java
 mapper.registerModule(new VavrModule());
```
But mapper is private.
```java
class Jackson2NodeFactory extends AbstractNodeFactory {
    private static final ObjectMapper mapper = new ObjectMapper();
```

**Comment by lukas-krecan** [comment]
Hi, thanks for feedback. Currently it is not possible to tune Jackson mapper, but you can serialize your object before comparison

```
assertThatJson(serialize(jsonObject1)).isEqualTo(serialize(jsonObject2))
```

where `serialize()` method takes the object and serializes it to String or to JsonNode tree.

I will try to figure out how to let people customize ObjectMapper in a maintainable and future-proof way.

**Comment by yarulan** [comment]
@lukas-krecan Totally, thank you!

**Comment by lukas-krecan** [comment]
Released a 2.3.0
